### ISSUE TITLE
need to update pypi version

### ISSUE THREAD

**Issue by radlws** [gtaylor/python-fedex#35]
I have to download using sudo pip install -U git+https://git@github.com/gtaylor/python-fedex.git

sudo pip install fedex gives me a version where I get this error while authenticating:

```
./generate_label.py
INFO:fedex:Using test server.
Traceback (most recent call last):
  File "./generate_label.py", line 22, in <module>
    shipment = FedexProcessShipmentRequest(CONFIG_OBJ)
  File "./lib/fedex/services/ship_service.py", line 37, in __init__
    *args, **kwargs)
  File "./lib/fedex/base_service.py", line 88, in __init__
    self.client = Client('file:///%s' % self.wsdl_path.lstrip('/'))
  File "./lib/suds/client.py", line 109, in __init__
    options.cache = ObjectCache(days=1)
  File "./lib/suds/properties.py", line 475, in __setattr__
    self.__pts__.set(name, value)
  File "./lib/suds/properties.py", line 284, in set
    self.provider(name).__set(name, value)
  File "./lib/suds/properties.py", line 424, in __set
    d.validate(value)
  File "./lib/suds/properties.py", line 184, in validate
    raise AttributeError,msg
AttributeError: "cache" must be: (<class suds.cache.Cache at 0x7fcd48ab2a78>,)
vagrant@ubuntu-vagrant-devbox:~/projects/gwt_fedex$ vim generate_label.py
vagrant@ubuntu-vagrant-devbox:~/projects/gwt_fedex$ ./generate_label.py
Traceback (most recent call last):
  File "./generate_label.py", line 14, in <module>
    from fedex_settings import CONFIG_OBJ
  File "/home/vagrant/projects/gwt_fedex/fedex_settings.py", line 24, in <module>
    freight_account_number='X10XXX02X', use_test_server=True)
TypeError: __init__() got an unexpected keyword argument 'freight_account_number'

```

**Comment by gtaylor** [comment]
What version of suds are you using?

And for the sake of clarification, are you saying that the latest git master is OK, but the current PyPi release isn't?

**Comment by radlws** [comment]
correct.


the pypi version is: fedex-1.0.14-py2.7.egg-info

which seems to be the prob

**Comment by gtaylor** [comment]
Released 1.1.0 on PyPi and tagged on here.

**Comment by radlws** [comment]
Thanks
